packages/rs-drive/src/drive/contract/mod.rs

```diff
@@ -95,6 +95,9 @@ mod tests {
     use crate::tests::helpers::setup::setup_drive_with_initial_state_structure;
     use dpp::version::PlatformVersion;
 
+    #[allow(dead_code)]
```

Comment:
Shall we just remove dead code?

Reply:
good question; I can (and would probably prefer to) but this feels less destructive / likely to be objected to by sam

Reply:
Yeah, but doing this we just shut linter which is not actually what you suppose to do

Reply:
I agree with Pasta.

Reply:
> shut linter which is not actually what you suppose to do

I do agree with this; but goal ATM is to get to a point where we can turn on CI enforcement on warnings as errors; as such need to suppress in cases like this